src/Core/src/Handlers/Label/LabelHandler.iOS.cs

```diff
@@ -39,9 +39,10 @@ public static void MapPadding(LabelHandler handler, ILabel label)
 			handler.TypedNativeView?.UpdatePadding(label);
 		}
 
-		static void MapFont(LabelHandler handler, ILabel label)
+		public static void MapFont(LabelHandler handler, ILabel label)
```

Comment:
Should the `MapFont*` also be removed above?

Reply:
Also fixed in https://github.com/dotnet/maui/pull/448

Reply:
Yeah, those other map methods keep popping back up after every rebase.